lib/modules/customToggles.js

```diff
@@ -50,6 +52,20 @@ function anyTogglesEnabled(toggles) {
 		.some(enabled => enabled);
 }
 
+function registerCommandLine() {
+	CommandLine.registerCommand(/^tog$|^toggle$/, ' tog[gle] [name] - toggles a custom toggle',
+		() => {},
+		(command, value) => {
+			const toggles = togglesByName()[value];
+			if (toggles) {
+				$(`#${toggles[0][2]}`).parent().toggleClass('enabled', toggleToggle(toggles));
```

Comment:
`onClickToggleMenuItem` can probably be used here.

Reply:
You were right, fixed!